retroshare-gui/src/gui/Posted/PostedCardView.cpp

```diff
@@ -309,8 +309,14 @@ void PostedCardView::fill()
 		// Wiping data - as its been passed to thumbnail.
 		
 		QPixmap sqpixmap = pixmap.scaled(desired_width,desired_height, Qt::KeepAspectRatioByExpanding, Qt::SmoothTransformation);
+		
+		QPixmap squaledpixmap = pixmap.scaled(640,480, Qt::KeepAspectRatio, Qt::SmoothTransformation);
+
+		if(pixmap.width() > 800) 
```

Comment:
Same issue as above:
- Different widths used
- Scaled outside of the if
- 480 for height is too small, we already have vertical scrollbar, so it can be larger. Or you can use scaledtowidth and not limit the height.

Reply:
You can remove this when you have a better solution, comic strips like which have more width then height its displayed it better